deurdu/README.md

```diff
@@ -1,3 +1,22 @@
 This directory contains all data and source code for deurdu blogging
 website.
 
+#For frontend or client side - to make it activate
+1) First install node.js in system
+Goto --> 'https://nodejs.org/en/download/source-code'
+
+2) Make sure to have XAMPP CONTROL PANEL
+
+3) After opeing the 'deurdu' folder
+    move to client using 'cd client'
+    type 'npm i' -- this will install all the packages with a single command
+    then type 'npm run dev' -- this will start project on localhost:5173
+
+============================================================================================================
+
+#For backend or server side - to make it activate
+1) Open another terminal on VSCode - make sure that client must be on another termianl and server must be on another termianl
+
+2) To install all the dependencies with a single command of server-side type 'npm i' in the server side terminal
+
+3) After that type 'nodemon index.js'.
```

Comment:
Trying this I got,
```bash
nodemon index.js 
[nodemon] 3.1.7
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,mjs,cjs,json
[nodemon] starting `node index.js`
/home/q550356/sample_code/deurdu/server/index.js:1
import express from 'express';
       ^^^^^^^

SyntaxError: Unexpected identifier
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
[nodemon] app crashed - waiting for file changes before starting...

```

Reply:
also why `nodemon` is not listed as dependency in server's `package.json` ?

Reply:
It is because in end i stated that : 
After downloading node, type **`npm i`** in server directory terminal and in result it will install all the dependencies and packages in project

Reply:
Please post your `node` version and we need to define a minimum version that users would need to work with the project. This version should be documented in README file.

```bash
#~/sample_code/deurdu/client$ node -v
v20.18.0
:~/sample_code/deurdu/client$ npm -v
10.8.2
:~/sample_code/deurdu/client$ 
```